beck-online.js

```diff
@@ -257,10 +257,10 @@ function scrapeBook(doc, url) {
 			contributorsAreNext = true;
 		}
 	}
-	var editors = doc.querySelectorAll('#titelseitetext .tpauthor');
-	for (let editor of editors) {
-		editor = authorRemoveTitlesEtc(editor.textContent);
-		item.creators.push(ZU.cleanAuthor(editor, creatorType));
+	var creators = doc.querySelectorAll('#titelseitetext .tpauthor');
+	for (let creator of creators) {
+		creator = authorRemoveTitlesEtc(creator.textContent);
+		item.creators.push(ZU.cleanAuthor(creator, creatorType));
```

Comment:
Ah, yes, that is good. I also thought about it, but then forgot to change in the end. BTW I didn't have an example for the author case and most (all?) books form beck-online might be edited.